pkg/apis/agones/v1/gameserver.go

```diff
@@ -150,6 +151,20 @@ type GameServerSpec struct {
 	SdkServer SdkServer `json:"sdkServer,omitempty"`
 	// Template describes the Pod that will be created for the GameServer
 	Template corev1.PodTemplateSpec `json:"template"`
+	// AlphaSpec describes the alpha properties for the GameServer
+	Alpha AlphaSpec `json:"alpha,omitempty"`
+}
+
+// AlphaSpec is the alpha properties of the GameServer
```

Comment:
nit: end with a period.

Also, maybe s/is/contains ?

Reply:
"contains" is good. Done.